operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -738,7 +738,7 @@ private void disableOnlineUpdateIfSpecChgNotCompatible(DomainPresenceInfo cached
       // and onlineUpdate, disable online update ??
 
       if (domainSourceType.equals(DomainSourceType.FromModel) && !isSpecChgOk4OnlineUpdate(liveInfo, cachedInfo)) {
-        LOGGER.fine("DomainType is FromModel and onlineUpdate is enabled, but there are changes in the domain "
+        LOGGER.info("DomainType is FromModel and onlineUpdate is enabled, but there are changes in the domain "
             + "spec incompatible for onlineUpdate, e.g. image name, serverPod environment, etc... resetting "
             + "onlineUpdate.enabled=false. The domain will roll after successful updates.");
```

Comment:
Is it necessary to set onlineUpdate.enabled to false? That seems a little confusing, and is something we'll need to document since (a) it's the only user controlled setting that can be reset, and (b) users are not likely to see this log message as this requires combing through the operator log.

Reply:
The method is currently not used, still trying to see what's the best way to determine if certain spec changed should not be using online update - most likely will change to error out, instead of changing it for the user.

Reply:
Any updates? I still see `o -> o.setEnabled(false)`.

Reply:
The method is not used,  working on changing it to error out.

Reply:
@tbarnes-us @rjeberhard   I have updated this section to basically not run the introspection and set the domain status with error message.  However, I am not sure how we can determine what is 'compatible' spec change for online update, and I am not sure there is an easy and comprehensive way.  Also,  the make right decision logic is getting more and more complicated, perhaps a get together and see better way of doing this?

Reply:
Work in progress in this area.